lib/github_changelog_generator/generator/generator_tags.rb

```diff
@@ -183,9 +183,9 @@ def filter_exact_tags(all_tags)
     end
 
     def warn_if_nonmatching_regex(all_tags)
-      unless all_tags.map { |t| t["name"] }.any? { |t| options[:exclude_tags] =~ t }
+      unless all_tags.map { |t| t["name"] }.any? { |t| /#{options[:exclude_tags_regex]}/ =~ t }
```

Comment:
@ldez I recently started building up my Ruby skill. I am wondering if double quote makes it clear?